src/app/services/comment-period.service.ts

```diff
@@ -108,15 +108,13 @@ export class CommentPeriodService {
   // get all comments associated with public comment period and map to comments attribute
   // return array of comment json objects
   private getCommentsByPCP(id) {
-    return this.api.getCommentsByPCPCode(id)
+    return this.api.getPublishedCommentsByPCPCode(id)
       .map((res: Response) => res.json())
       .map((pcpComments) => {
         pcpComments.forEach((comment, index) => {
-          if (comment.isPublished) {
```

Comment:
Remove the filtering here now that the api actually only returned published comments (see comment from: https://github.com/bcgov/esm-server/pull/420/files).